moped-editor/src/components/GridTable/FiltersCommonOperators.js

```diff
@@ -107,39 +107,46 @@ export const FiltersCommonOperators = {
   },
   date_equals: {
     operator: "_eq",
-    label: "is",
+    label: "on",
     description: "The date is equal to.",
     envelope: null,
     type: "date",
   },
   date_greater_than: {
     operator: "_gt",
-    label: "greater than",
-    description: "The date is greater than.",
+    label: "after",
+    description: "The date is after.",
```

Comment:
i couldnt figure out what these descriptions are used for, maybe just for us? in that case maybe i didnt need to update these descriptions idk